mitodl/ocw-studio
# update poetry lock file

# What are the relevant tickets?
N/A

# Description (What does it do?)
Recently https://github.com/mitodl/ocw-studio/pull/1930 was merged to update the pre-commit and linting configuration in `ocw-studio`. It caused some issues with poetry as well as running pre-commit locally. This PR implements some fixes:

 - `poetry lock && poetry install` was run to update the `poetry.lock` file
 - `shfmt` was removed from the `pre-commit` configuration for now as it throws an error when you run it locally

# How can this be tested?
If you rebase a PR off of this branch, Github Actions should run successfully. You should also be able to make a commit to your branch locally without pre-commit throwing an error.
